### PR TITLE
[ADD] l10n_pe: New types of identity documents from sunat

### PR DESCRIPTION
**Pull request by desdelinux**
By Sunat regulations from 08/01/2021 two new types of identity documents will be accepted: Temporary Residence Permit and Safe Passage

Legal Reference:

Website:  https://cpe.sunat.gob.pe/node/88
Document with Catalog Update No. 8: https://cpe.sunat.gob.pe/sites/default/files/inline-files/AjustesValidacionesCPEv20210727%20%28ultimo%29.xlsx

### CODE REVIEW

**Review by nhomar — APPROVED** [review]

**Comment by vbe-odoo** [comment]
@jco-odoo this is ready for technical review.
2 new identification types are added for the partner_id > this is part of the changes needed for next week > legal requirement.
Best

File: addons/l10n_pe/i18n/es.po [hunk 1]
```diff
@@ -158,7 +158,12 @@ msgstr "IVAP - Impuesto a la Venta Arroz Pilado"
 #. module: l10n_pe
 #: model:l10n_latam.identification.type,name:l10n_pe.it_IDCR
 msgid "Identity document of the country of residence"
-msgstr "Documento de identidad del país de recidencia"
+msgstr "Documento de identidad del país de residencia"
```
Comment (jco-odoo) [inline addons/l10n_pe/i18n/es.po:161]: If you change the .po, please change the .pot as well please.

**Review by jco-odoo — COMMENTED** [review]
Adding an identification type, should not give any problems, so ok, but please check the details about the translation.

**Comment by jco-odoo** [comment]
@desdelinux Would be nice if you could still squash the commits

**Comment by vbe-odoo** [comment]
Hi @nhomar @desdelinux can you do the squash? Thanks!
cc @jco-odoo

**Comment by jco-odoo** [comment]
@nhomar @desdelinux I see that Luigys should also sign the CLA.  In the meantime, I made a new PR here: #74403

**Comment by desdelinux** [comment]
> @nhomar @desdelinux I see that Luigys should also sign the CLA. In the meantime, I made a new PR here: #74403

@jco-odoo Thanks for the new pr,  I have signed the cla here https://github.com/odoo/odoo/pull/74343, but it has not yet been approved

**Comment by jco-odoo** [comment]
@desdelinux Let's get it approved then.

**Comment by vbe-odoo** [comment]
Hi @jco-odoo Are we moving forward with this PR or are we moving forward with the PR https://github.com/odoo/odoo/pull/74403

Because I see these 2 PRs open, and not sure which one will be the one we will merge and which one we should close.
I can see that the CLA is already approved by you.

Thanks!
cc' @desdelinux

**Pull request closed without merging**